presidio-analyzer/analyzer/predefined_recognizers/us_bank_recognizer.py

```diff
@@ -9,11 +9,11 @@
     "bank"
     # Task #603: Support keyphrases: change to "checking account"
     # as part of keyphrase change
-    "checking",
```

Comment:
is this relevant to this PR?

Reply:
Yes. I had to change the demo website text because spaCy now identifies new things (false positives). By changing this, I've encountered other issues with other recognizers...